ketcher-autotests/tests/User-Interface/Clear-Canvas/clear-canvas.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  selectTopPanelButton,
+  delay,
+} from '@utils/canvas';
+import { clickInTheMiddleOfTheScreen, pressButton } from '@utils/clicks';
+import { waitForPageInit } from '@utils/common';
+import { openFileAndAddToCanvas } from '@utils/files';
+import { DELAY_IN_SECONDS } from '@utils/globals';
+import { RingButton, TopPanelButton, selectRing } from '@utils/selectors';
+import { addTextBoxToCanvas } from '@utils/selectors/addTextBoxToCanvas';
+
+test.describe('Clear canvas', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test('Clear Canvas - "Clear canvas" button', async ({ page }) => {
+    // Test case:EPMLSOPKET-1702
+    // Checking apperance of the tooltip and deleting object with Clear canvas button
+    // await selectTopPanelButton(TopPanelButton.Clear, page);
+    await page.getByRole('button', { name: 'Clear Canvas (Ctrl+Del)' }).hover();
+    await page.getByTitle('Clear Canvas (Ctrl+Del)').click();
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('12345');
+    await pressButton(page, 'Apply');
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Clear Canvas - Undo/Redo', async ({ page }) => {
+    // Test case: EPMLSOPKET-1704
+    // Checking deleting object with Clear canvas button with funcion of Undo/Redo buttons
+    await openFileAndAddToCanvas('reaction-dif-prop.rxn', page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Clear Canvas - Structure is opened from ket-file', async ({ page }) => {
+    // Test case:EPMLSOPKET-1705
+    // Checking deleting object with Clear canvas button with funcion of hotkyes (Ctrl+ Del)
+    await openFileAndAddToCanvas('ketcher.ket', page);
```

Comment:
Why do you perform multiple actions in test? You write
"Checking deleting object with Clear canvas button with funcion of hotkyes (Ctrl+ Del)"
What do you check? Do you check that canvas is cleared after clicking the button in top panel? Or do you check that canvas is cleared after shortcut?
If you want to check both, then you need to create 2 separate tests